lib/fog/compute/google/requests/insert_server.rb

```diff
@@ -129,12 +129,22 @@ def handle_networks(options)
           end
 
           # Objectify the network if needed
-          unless network.is_a? Network
+          unless network.is_a?(Network)
             network = networks.get(network)
+            network_interfaces = network.get_as_interface_config(access_config)
+          end
+
+          if options.key? "subnetwork"
+            subnetwork = options.delete "subnetwork"
+            # Objectify the subnetwork if needed
+            unless subnetwork.is_a?(Subnetwork)
+              subnetwork = subnetworks.get(subnetwork, region_name)
+              network_interfaces = subnetwork.update_interface_config(network_interfaces)
+            end
           end
 
           # Return a networkInterfaces array
-          [network.get_as_interface_config(access_config)]
+          [network_interfaces]
```

Comment:
Could you tell me please where I should put google_project arguments? I have created the file ~/.fog with credentials as described in README.md but I get the following error 
```
lkuntar@lkuntar-VirtualBox:~/workspace/forks/fog-google$ rake test TEST="test/**/test_compute_networks_collection.rb"
Run options: --seed 41630

# Running:

E

Finished in 0.004641s, 215.4935 runs/s, 0.0000 assertions/s.

  1) Error:
TestComputeNetworksCollection#test_network_workflow:
ArgumentError: Missing required arguments: google_project
    /var/lib/gems/2.2.0/gems/fog-core-1.43.0/lib/fog/core/service.rb:244:in `validate_options'
    /var/lib/gems/2.2.0/gems/fog-core-1.43.0/lib/fog/core/service.rb:268:in `handle_settings'
    /var/lib/gems/2.2.0/gems/fog-core-1.43.0/lib/fog/core/service.rb:98:in `new'
    /home/lkuntar/workspace/forks/fog-google/test/integration/compute/test_compute_networks_collection.rb:17:in `test_network_workflow'

1 runs, 0 assertions, 0 failures, 1 errors, 0 skips
/var/lib/gems/2.2.0/gems/fog-core-1.43.0/lib/fog/core/service.rb:244:in `validate_options': Missing required arguments: google_project (ArgumentError)
	from /var/lib/gems/2.2.0/gems/fog-core-1.43.0/lib/fog/core/service.rb:268:in `handle_settings'
	from /var/lib/gems/2.2.0/gems/fog-core-1.43.0/lib/fog/core/service.rb:98:in `new'
	from /home/lkuntar/workspace/forks/fog-google/test/integration/compute/test_compute_networks_collection.rb:11:in `block in <class:TestComputeNetworksCollection>'
	from /var/lib/gems/2.2.0/gems/minitest-5.10.1/lib/minitest.rb:58:in `call'
	from /var/lib/gems/2.2.0/gems/minitest-5.10.1/lib/minitest.rb:58:in `reverse_each'
	from /var/lib/gems/2.2.0/gems/minitest-5.10.1/lib/minitest.rb:58:in `block (2 levels) in autorun'
rake aborted!
Command failed with status (1)
/var/lib/gems/2.2.0/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => test

```